sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/spark/udf/GetFeedRangeForPartitionKeyValues.scala

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.cosmos.spark.udf
+
+import com.azure.cosmos.implementation.SparkBridgeImplementationInternal
+import com.azure.cosmos.spark.CosmosPredicates.requireNotNullOrEmpty
+import org.apache.spark.sql.api.java.UDF2
+
+
+@SerialVersionUID(1L)
+class GetFeedRangeForPartitionKeyValues extends UDF2[String, String, String] {
```

Comment:
Currently, we have a UDF called` GetFeedRangeForPartitionKeyValue extends UDF2[String, Object, String]` 

I am wondering should we rename this udf to make it more obvious that it is for hierarchical partition keys or we combine these two udf into one, because I think the logic here also applies to the hash partition key
@FabianMeiswinkel  @TheovanKraay